src/__mocks__/datasource.ts

```diff
@@ -8,6 +8,7 @@ export const mockDatasource = new DataSource({
   type: 'redshift-datasource',
   name: 'Redshift Data Source',
   jsonData: {},
+  access: 'direct',
```

Comment:
what does this do again?

Reply:
not really sure, think the type just expects an access property.